introspection-engine/core/src/rpc.rs

```diff
@@ -86,13 +101,28 @@ impl RpcImpl {
                 )))
             }
             Ok(introspection_result) => {
+                let warnings = match could_not_parse_input_data_model {
+                    true if reintrospect => {
+                        let mut warnings = introspection_result.warnings;
+                        warnings.push(Warning {
+                        code: 0,
+                        message:
+                        "The input datamodel could not be parsed. This means it was not used to enrich the introspected datamodel with previous manual changes."
```

Comment:
Depending on how the CLI handles it, we could return the whole parser error message with colors and position information here in the future.